pkg/reconciler/workload/heartbeat/heartbeat_manager.go

```diff
@@ -24,7 +24,7 @@ import (
 
 	workloadv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/workload/v1alpha1"
 	"github.com/kcp-dev/kcp/pkg/reconciler/workload/basecontroller"
-	conditionsv1alpha1 "github.com/kcp-dev/kcp/third_party/conditions/apis/conditions/v1alpha1"
+	conditionsapi "github.com/kcp-dev/kcp/third_party/conditions/apis/conditions/v1alpha1"
```

Comment:
Not super important, but curious why the change here?

Reply:
The `conditions` library uses `conditionsapi` as the conventional import name. I'd been spelunking in the lib and was going for consistency with it. I now see `conditionsv1alpha1` is used in KCP, though, so I'll switch to that.

Reply:
I think we have a mix here & there. Either is fine for now.